langkit/emitter.py

```diff
@@ -695,6 +695,12 @@ def emit_ocaml_api(self, ctx):
                 ocaml_api=ctx.ocaml_api_settings
             )
 
+            # Write an empty ocamlformat file so we can call ocamlformat
```

Comment:
Actually, it works. Not sure why it did not worked with the version I sent you... I probably manually called ocamlformat only on the .ml. I still moved this code since it is weidly placed.

Reply:
Great!